natsstreaming/nats_streaming.go

```diff
@@ -152,11 +153,18 @@ type AsyncMessageSourceConfig struct {
 	MaxInFlight int
 	AckWait     time.Duration
 	Offset      int64
+
+	// number in seconds between pings (min 1)
+	ConnectionPingInterval int
```

Comment:
I would check that it's at least 1 in the`NewAsyncMessageSource ` function or use the default value if it's 0 and return an error otherwise.